upgrading/topics/install_new_version-quarkus.adoc

```diff
@@ -0,0 +1,14 @@
+[[_install_new_version]]
+
+== Upgrading the {project_name} server
+
+It is important that you upgrade {project_name} server before upgrading the adapters.
+
+.Prerequisites
+* Handle any open transactions and delete the data/tx-object-store/ transaction directory.
+
+.Procedure
+. Download the new server archive
+. Move the downloaded archive to the desired location.
+. Extract the archive. This step installs a clean instance of the latest {project_name} release.
+. Copy `conf/`, `providers/` and `themes/` from the previous installation to the new installation.
```

Comment:
And make sure to run the `build` command to update the server configuration. Not sure if that is implicit though.